public/images/illustrations/illustration-500.svg

```diff
@@ -1 +1 @@
.3.1z"/><path class="st2" d="M102.7 70.8h-.4c-.3 0-.5-.3-.5-.5 0-.3.3-.5.5-.5.6 0 1.1 0 1.6-.2.3-.1.5 0 .6.3s0 .5-.3.6c-.3.2-.9.3-1.5.3zm-2.1-7.7c.3-.1.7.2.7.5l.2 1.1c.1.4-.2.7-.5.8-.3.1-.7-.2-.7-.5l-.2-1.1c-.1-.4.1-.8.5-.8z"/><path class="st4" d="M288.7 86.6c-5.8.9-13.6 29.7-7.8 34.1 5.1 3.8 13.6-4.4 18-8.9l-3.8-5.2-2.6-.1"/><path class="st6" d="M315.5 77.2c.6 2.6 1 5.2 1.1 7.8.2 2.7 0 5.3-.5 7.9l-.5 2c-.2.6-.5 1.3-.8 1.9-.6 1.2-1.5 2.3-2.5 3.2-2.1 1.8-4.8 2.7-7.6 2.7-2.7.2-5.4-.1-8-.9-1.3-.4-2.6-1-3.7-1.8s-2-1.9-2.7-3.1c-1.2-2.4-1.8-5.1-1.6-7.8.1-2.6.6-5.3 1.3-7.8l.2-.1.1.2c-.5 2.6-.7 5.1-.8 7.7-.1 2.5.4 5.1 1.6 7.3.6 1.1 1.4 2 2.4 2.8 1 .7 2.2 1.3 3.4 1.6 2.5.7 5 .9 7.6.8 2.5 0 4.9-.9 6.9-2.5.9-.8 1.7-1.8 2.3-2.9l.7-1.7c.2-.6.4-1.2.5-1.8.5-2.5.7-5.1.5-7.7-.1-2.6-.5-5.1-.7-7.8 0-.2.1-.4.3-.4.3.2.5.3.5.4z"/><path class="st2" d="M272.3 60.5l-7.7-9.6-.3-.3.4-.1c3-.5 6.1 0 8.9 1.2 2.8 1.3 5.1 3.3 6.8 5.9 1.6 2.6 2.4 5.6 2.2 8.7-.2 3-1 6-2.3 8.7v-.4l21.1 26.5 21.1 26.6-.7-.1c2.6-1.5 5.6-2.2 8.6-1.9 3 .4 5.7 1.9 7.7 4.2 2 2.2 3.4 4.9 3.9 7.8.5 2.9.2 5.9-.8 8.7l-.3.9-.6-.7-7.7-9.6.5.2-8.7 1.3.4-.3-3.2 8.2-.1-.5 7.7 9.6.6.7-.9.1c-2.9.4-5.8.1-8.5-1s-5-2.9-6.7-5.3c-1.8-2.3-2.7-5.2-2.6-8.2.3-2.9 1.5-5.7 3.4-7.9v.7l-21.2-26.9-10.6-13.5-10.6-13.5.4.2c-2.9.6-5.9.6-8.8.1-3-.6-5.7-2-7.8-4.2-2.1-2.2-3.4-4.9-4-7.8s-.3-6 .8-8.8l.2-.5.3.4 7.6 9.7-.2-.1 8.7-1.2-.1.1 3.1-8.1z"/><path class="st5" d="M292.6 106.8l-3.9 2.1-3.8 2.1-.2-.1v-.2c1-1.1 2.2-2.1 3.6-2.7 1.3-.7 2.7-1.3 4.1-1.8.2 0 .3.1.4.2 0 .2 0 .4-.2.4z"/><path class="st0" d="M294 106.6c2.7-4.4 5.7-6.8 10.2-6.8s4.2 4.8 2.1 5.4c-3.7 1-2.5 3.7-6 5.9l-2.4 1.4-3.9-5.9z"/><path class="st5" d="M306.2 105.4c-.5 0-1 .1-1.5.4-.4.3-.7.7-1 1.1-.5 1-1.1 2-1.8 3s-1.7 1.8-2.8 2.3c-1 .5-1.9 1.1-2.7 1.9-1.7 1.5-3.2 3.2-5.1 4.6-.9.7-1.9 1.4-3 1.9s-2.3.7-3.4.8l-.2-.2.2-.2c1.1-.2 2.2-.5 3.2-1s1.9-1.2 2.8-1.9c1.7-1.5 3.2-3.2 5-4.7.9-.8 1.9-1.5 2.9-2s1.9-1.1 2.5-2c.7-.8 1.2-1.9 1.8-2.9.3-.5.8-1 1.3-1.3.6-.3 1.2-.4 1.8-.4.2 0 .3.2.2.4.1.1 0 .2-.2.2zm1.2-2.4c-.7-.1-1.3-.1-2 0s-1.3.2-2 .4l-1.9.6c-.6.2-1.3.4-2 .6l-.2-.1v-.1c1.1-.9 2.4-1.5 3.8-1.9.7-.2 1.4-.3 2.1-.4s1.5 0 2.2.2c.2.1.2.2.2.4s-.1.2-.2.3zm-1.9-2.3c-1.7-.1-3.5.2-5.1.8-.8.3-1.5.7-2.2 1.2s-1.3 1.1-1.8 1.8h-.2l-.1-.2c.3-.9.9-1.7 1.6-2.3.7-.6 1.5-1.1 2.4-1.4.9-.4 1.8-.6 2.7-.7.9-.1 1.9-.1 2.8.1.2 0 .3.2.2.4 0 .2-.1.3-.3.3zm9.3 1.5c1 3.1 1.3 6.4.9 9.6-.1 1.6-.4 3.2-.7 4.8-.3 1.6-.8 3.1-1.4 4.6l-.2.1-.1-.2.9-4.7c.3-1.6.5-3.1.6-4.7s.2-3.1.1-4.7-.3-3.1-.7-4.6c0-.2.1-.3.2-.4.2 0 .4.1.4.2z"/><path class="st0" d="M302.2 136.2c-3.2 0-6.6-.2-8.3-.5-.4-.1-.6-.5-.6-.9.1-.4.5-.6.9-.6 3.2.6 13.1.7 15.3 0 .4-.1.8.1.9.5s-.1.8-.5.9c-1.2.4-4.4.6-7.7.6z"/><path class="st1" d="M40.2 215.1c26.6-.6 53.3-.7 79.9-.8l79.9-.1c53.3.1 106.6.1 159.8.7.2 0 .4.2.4.4s-.2.4-.4.4c-53.3.7-106.6.6-159.8.7l-79.9-.1c-26.6-.1-53.3-.3-79.9-.8-.1 0-.2-.1-.2-.2s.1-.2.2-.2zM192 102c-12.7 0-23 10.3-23 23s10.3 23 23 23 23-10.3 23-23-10.3-23-23-23zm2.3 34.5h-4.6v-4.6h4.6v4.6zm0-9.2h-4.6l-1.2-13.8h6.9l-1.1 13.8z"/></svg>
```

Comment:
That SVG has some cruft right now. I can come through and clean it up, otherwise maybe just pass it through https://jakearchibald.github.io/svgomg/

Reply:
Be my guest please 😉 I'd probably remove the `id` and the `<title>`, but not much more other than that.